app/main.py

```diff
@@ -1,8 +1,50 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self,
+                 distance_from_city_center: float,
+                 clean_power: int,
+                 average_rating: float,
+                 count_of_ratings: int) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: list) -> float:
+        total_cost = []
+        for car in cars:
+            if car.clean_mark < self.clean_power:
+                total_cost.append(self.calculate_washing_price(car))
+                self.wash_single_car(car)
+            else:
+                total_cost.append(0)
```

Comment:
Zauważ, że mamy tutaj to samo sprawdzenie, które występuje w funkcji `wash_single_car`, tak więc jest to powtórzenie kodu. Aby się tego pozbyć, mógłbyś przykładowo zmienić funkcję `wash_single_car`, aby zwracała typ `bool`, `True` - jeżeli auto zostało umyte oraz `False` - jeżeli nie zostało umyte, w ten sposób można zmienić tę funkcję pomijając powielenie warunków:
`if self.wash_single_car(car):
      total_cost.append(self.calculate_washing_price(car))
`
W przypadku, gdy auto nie zostanie umyte nie musisz dodawać wartości `0` do total_cost ponieważ nie zmieni to w żaden sposób wyniku, tak więc blok `else` jest w tym przypadku niepotrzebny.